cmd/main.go

```diff
@@ -613,7 +617,7 @@ func resolveKubeConfigPath(opt *Options) error {
 
 func startMCPServer(ctx context.Context, opt Options) error {
 	workDir := filepath.Join(os.TempDir(), "kubectl-ai-mcp")
-	if err := os.MkdirAll(workDir, 0755); err != nil {
+	if err := os.MkdirAll(workDir, 0o755); err != nil {
```

Comment:
Can we preserve the original permissions ?

Reply:
Editor added that, sorry. Reverting.